### PR TITLE
[ci skip] Add issue and pr template

### LINKED ISSUE #3432: Investigate "issue template"
According to https://github.com/dear-github/dear-github/issues/125, it looks like issue templates are now silently supported. We should have one.

### PR DESCRIPTION
**Pull request by azakus**
### Description
Add issue and pr template

#### Reference issue
Fixes #3432

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+#### What you expected
+
+
+#### What you saw
+
+
+### Live Demo
+http://jsbin.com/…
```
Comment (samccone) [inline issue_template.md]: can we link to the prefilled template?

Comment (azakus) [inline issue_template.md]: Yes, @kevinpschaaf suggested the same thing.

**Comment by abdonrd** [comment]
LGTM! Nice work @azakus! :)

**Comment by azakus** [comment]
Heard @kevinpschaaf say the words "Looks" and "good" relatively closely to each other.

**Pull request merged**